%YAML:1.0
currentTableZ:
   - 1.7006761821037295e-01
cropUpperLeftCorner:
   - 320.
   - 200.
vanishingPointReticle:
   - 311.
   - 163.
heightReticles:
   - 319.
   - 326.
   - 330.
   - 346.
   - 137.
   - 132.
   - 118.
   - 98.
colorReticles:
   - 462
   - 450
   - 439
   - 428
   - 419
   - 410
   - 405
   - 399
   - 394
   - 389
   - 383
   - 381
   - 379
   - 378
   - 153
   - 153
   - 153
   - 153
   - 153
   - 154
   - 154
   - 154
   - 154
   - 154
   - 155
   - 155
   - 155
   - 155
lensCorrections:
   - 6.1999999999999944e-01
   - 5.4798737571361678e-01
   - 5.1462155816541522e-01
   - 5.0999999999999956e-01
   - 9.0991915917454691e-01
   - 9.3756658804408077e-01
   - 9.2000000000000004e-01
   - 9.3000000000000005e-01
gear0offset:
   - 2.9999999329447746e-02
   - 2.3000000044703484e-02
   - 1.6722800210118294e-02
   - 0.
